torch/quantization/quantize_jit.py

```diff
@@ -83,20 +97,17 @@ def convert_dynamic_jit(model, inplace=False, debug=False):
     return _convert_jit(model, inplace, debug, quant_type=QuantType.DYNAMIC)
 
 def _quantize_jit(model, qconfig_dict, run_fn=None, run_args=None, inplace=False, debug=False, quant_type=QuantType.STATIC):
-    assert not inplace, "We don't support inplace right now"
     # Always do inplace convert because the Tensor is already
     # copied in prepare_jit when inplace is False
     if quant_type == QuantType.DYNAMIC:
         model = prepare_dynamic_jit(model, qconfig_dict, inplace)
-        # TODO: change inplace to True
-        model = convert_dynamic_jit(model, False, debug)
+        model = convert_dynamic_jit(model, True, debug)
     else:
         assert run_fn, "Must provide calibration function for post training static quantization"
         assert run_args, "Must provide calibration dataset for post training static quantization"
         model = prepare_jit(model, qconfig_dict, inplace)
```

Comment:
If inplace=True, why is prepare_jit returning a model? Should we have it not return anything if inplace=True?

Reply:
this is just a convenience for us that we return the model so that we don't need to have if branch here. Also this is the same as eager mode quantization API: https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L150